pkg/services/ngalert/api/generated_base_api_provisioning.go

```diff
@@ -46,7 +46,9 @@ type ProvisioningApi interface {
 func (f *ProvisioningApiHandler) RouteDeleteAlertRule(ctx *models.ReqContext) response.Response {
 	// Parse Path Parameters
 	uIDParam := web.Params(ctx.Req)[":UID"]
-	return f.handleRouteDeleteAlertRule(ctx, uIDParam)
+	// Get Header
+	disablingProvenanceHeader := ctx.Req.Header.Get("x-disable-provenance")
```

Comment:
If you want you can use a const for the header name

Reply:
Sure!